tests/unit_tests/commandline/test_geoips_validate.py

```diff
@@ -22,18 +22,19 @@ def all_possible_subcommand_combinations(self):
         if not hasattr(self, "_cmd_list"):
             self._cmd_list = []
             base_args = self._validate_args
-            pkg_path = str(resources.files("geoips_clavrx") / "plugins")
-            # validate all plugins from package geoips_clavrx
-            for plugin_type in ["modules", "yaml"]:
-                if plugin_type == "modules":
-                    plugin_path_str = f"{pkg_path}/{plugin_type}/**/*.py"
-                else:
-                    plugin_path_str = f"{pkg_path}/{plugin_type}/**/*.yaml"
-                plugin_paths = sorted(glob(plugin_path_str, recursive=True))
-                for plugin_path in plugin_paths:
-                    self._cmd_list.append(base_args + [plugin_path])
-            # Add argument list to retrieve help message
-            self._cmd_list.append(base_args + ["-h"])
+            for plugin_package in self.plugin_packages:
+                pkg_path = str(resources.files(plugin_package) / "plugins")
+                # validate all plugins from package geoips_clavrx
+                for plugin_type in ["modules", "yaml"]:
+                    if plugin_type == "modules":
+                        plugin_path_str = f"{pkg_path}/{plugin_type}/**/*.py"
+                    else:
+                        plugin_path_str = f"{pkg_path}/{plugin_type}/**/*.yaml"
+                    plugin_paths = sorted(glob(plugin_path_str, recursive=True))
+                    for plugin_path in plugin_paths:
+                        self._cmd_list.append(base_args + [plugin_path])
+                # Add argument list to retrieve help message
+                self._cmd_list.append(base_args + ["-h"])
```

Comment:
I am assuming this is what was intended for the "all_possible_subcommand_combinations - and not only running in the "geoips_clavrx" package?  Let me know if this was not the intent.

Reply:
I chose to run on geoips_clavrx because it is a fairly simple and small package. We should probably change the name from "all_possible_subcommand_combinations" to "selected_subcommand_combinations" or something along those lines. Some unit tests are stochastic in the command combinations they will test and we should make that variable reflect that ideology.